epPlex/epPlex
# WIP: Create collection system

How to test
1) Fix paths that I changed to fzzyyti
2) Run these
```bash
anchor localnet
anchor run test-collection
```
Two tests should pass. The test will create a new collection with a T22 collection NFT. The mint will be signed by a PDA.
Next, it will create 10 mints for the collection, all signed with a PDA.

Minimum to do before merging
1) Make sure a non-collection mint works
2) Fix tests in the SDK
3) Verify what to do with the mint authority on the tokens
4) Remove dead code
5) Tighten up account constraints
